examples/demo-rollup/src/main.rs

```diff
@@ -26,12 +26,41 @@ use sov_rollup_interface::stf::StateTransitionFunction;
 use sov_state::Storage;
 use std::env;
 use std::net::SocketAddr;
+use std::sync::Arc;
+use tokio::sync::RwLock;
 use tracing::Level;
 use tracing::{debug, info};
 
 // RPC related imports
 use demo_stf::app::get_rpc_methods;
+use risc0_adapter::Risc0MethodId;
 use sov_modules_api::RpcRunner;
+use sov_rollup_interface::services::batch_builder::BatchBuilder;
+use sov_rollup_interface::zk::traits::{Zkvm, ZkvmHost};
+
+/// In order to allow DemoAppRunner to be `Send`, this Zkvm stub is added
+/// It can be used to communicate with a Risc0Host over socket.
+struct RemoteZkHost {
+    _address: SocketAddr,
+}
```

Comment:
I think we can remove this impl entirely if we add a `DemoAppRunner` implementation for the `Standard` config. 

IIRC, we require the NativeAppRunner to have access to a ZkvmHost (meaning that it can verify proofs *and* write data to the guest). But in reality, the only requirement for the block builder is that it can verify proofs. 

To get rid of this ugly remote zk host, I think we just need to rename `NativeAppRunner` to `ProverAppRunner` and add a `StandardAppRunner` which uses just a regular `Risc0Verifier` as its Zkvm. We'll also need to implement a `Risc0Verifier`, but that should be very very simple.

Reply:
Done!